src/Block.php

```diff
@@ -0,0 +1,99 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Nexylan packages.
+ *
+ * (c) Nexylan SAS <contact@nexylan.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Nexy\Slack;
+
+/**
+ * @author Sullivan Senechal <soullivaneuh@gmail.com>
+ * @author Mikey McLellan <mikey@mclellan.org.nz>
+ */
+class Block
+{
+    /**
+     * @var string
+     */
+    protected $type;
+
+    /**
+     * @var string
+     */
+    private $blockId;
+
+    public function __construct(?string $type)
```

Comment:
Missing PHPDoc

Reply:
Not necessary, the strict typing is enough.

Except if you want to add detail about `$type` description.